api/logger.proto

```
@@ -0,0 +1,82 @@
+// Copyright © 2025 Prabhjot Singh Sethi, All Rights reserved
+// Author: Prabhjot Singh Sethi <prabhjot.sethi@gmail.com>
+
+syntax = "proto3";
+
+package api;
+
+import "google/api/annotations.proto";
```

Buf (1.57.2)

import "google/api/annotations.proto": file does not exist

(COMPILE)